drivers/i2c/Kconfig.sf32lb

```diff
@@ -7,3 +7,13 @@ config I2C_SF32LB
 	depends on DT_HAS_SIFLI_SF32LB_I2C_ENABLED
 	help
 	  Enable I2C driver for SiFli SF32LB.
+
+if I2C_SF32LB
+
+config I2C_SF32LB_INTERRUPT
+	bool "SF32LB I2C Interrupt Support"
+	default y
+	help
+	  Enable interrupt-driven transfers for the SF32LB I2C driver.
```

Comment:
why would one set this to no though? couldn't you drop the non interrupt code if this is strictly better?

Reply:
just keep it, It seems that gmarull still maintain some code base on this